pkg/server.go

```diff
@@ -280,6 +282,19 @@ func (s *gitClient) getClient(ctx context.Context) (opt *gitOptions, err error)
 
 		insecure := store.Properties["insecure"] == "true"
 
+		var timeout time.Duration
+		if timeoutStr, exists := store.Properties["timeout"]; exists {
+			timeout, err = time.ParseDuration(timeoutStr)
+			if err != nil {
+				err = errors.New("fail to parse timeout")
+			}
+		}
+
+		customClient := &netHttp.Client{
+			Timeout: timeout,
```

Comment:
What will happen if no timeout is given? How about give it a default value?

Reply:
I give a default value of 60s